client/components/deposits-overview/next-deposit.tsx

```diff
@@ -53,6 +53,22 @@ const NextDepositDetails: React.FC< NextDepositProps > = ( {
 
 	return (
 		<>
+			{ /* Next Deposit Heading */ }
+			<CardBody className="wcpay-deposits-overview__heading">
+				<span className="wcpay-deposits-overview__heading__title">
+					<Loadable
+						isLoading={ isLoading }
+						value={ strings.nextDeposit.title }
+					/>
+				</span>
+
+				<span className="wcpay-deposits-overview__heading__description">
+					<Loadable
+						isLoading={ isLoading }
+						value={ strings.nextDeposit.description }
+					/>
+				</span>
+			</CardBody>
```

Comment:
This might also be a merge conflict or copied from another commit. This addition is duplicating the next deposit heading. 

<img width="706" alt="Screenshot 2023-04-13 at 5 16 31 pm" src="https://user-images.githubusercontent.com/8490476/231683074-874f6982-502a-4804-8906-e6a329f82143.png">

Reply:
Spot on - this is a merge merge conflict conflict. I've merged this and fixed it twice now (same thing happened on other branch!!)